sdcm/cluster.py

```diff
@@ -706,6 +706,8 @@ def _notify_backtrace(self, last):
         :param last: Whether to show only the last backtrace.
         """
         result = self._get_coredump_backtraces(last=last)
+        if 'command not found' in result.stderr:
```

Comment:
better to check exit code instead of parsing output

Reply:
https://www.tldp.org/LDP/abs/html/exitcodes.html
check for 127

Reply:
Yes, it's better. Current one coredumpctl command is that:
`n_backtraces_cmd = 'sudo coredumpctl --no-pager --no-legend 2>&1 ||true'`

Firstly we need to remove `true` to return right exit_status of coredumpctl.
`n_backtraces_cmd = 'sudo coredumpctl --no-pager --no-legend 2>&1'`